web/ui/react-app/src/pages/targets/JobsList.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { FC } from 'react';
+import { FilterData } from './Filter';
+import { useFetch } from '../../utils/useFetch';
+import { TargetGroup, groupTargets } from './target';
+import JobPanel from './JobPanel';
+import Error from '../../ErrorAlert';
+import Loader from '../../Loader';
+import PathPrefixProps from '../../PathPrefixProps';
+
+interface JobsListProps {
+  filter: FilterData;
+}
+
+const filterByHealth = ({ metadata, targets }: TargetGroup, { showHealthy, showUnhealthy }: FilterData): boolean => {
+  const isHealthy = metadata.up === targets.length;
+  return (isHealthy && showHealthy) || (!isHealthy && showUnhealthy);
+};
+
+const JobsList: FC<JobsListProps & PathPrefixProps> = ({ filter, pathPrefix }) => {
```

Comment:
Although there's often a 1:1 relation between jobs and scrape configs / scrape pools, it can also be 1:n or m:1 or even n:m in some cases. So technically it would be more correct to call it a `ScrapePoolList` or something like that.

Btw., in the future we might want to have arbitrary labels by which we can group things on the /targets page, but let's think of that when we get there.

Reply:
This is now `ScrapePoolList` which contains a number of `ScrapePoolPanel`s.

I think that the `groupTargets` function could be extended (in a future pr) to enable different groupings.